### PR TITLE
fix python-wheel ptest

### PR DESCRIPTION
**Pull request by 0xba1a**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Make PTest of python-wheel to run successfully

###### Change Log  

- BR multiple python modules required by PTest
- pip install other modules which are not provided by Mariner RPMs

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

Local build

### CODE REVIEW

File: SPECS/python-wheel/python-wheel.spec [hunk 1]
```diff
@@ -85,7 +89,8 @@ install -p dist/%{python_wheelname} -t %{buildroot}%{python_wheeldir}
 
 %check
 rm setup.cfg
-PYTHONPATH=%{buildroot}%{python3_sitelib} py.test-3 -v --ignore build
+%{python3} -m pip install pluggy more-itertools
+PYTHONPATH=%{buildroot}%{python3_sitelib} py.test3 -v --ignore build
```
Comment (MateuszMalisz) [inline SPECS/python-wheel/python-wheel.spec:93]: Is there a difference between this and `%{python3}`?

File: SPECS/python-wheel/python-wheel.spec [hunk 2]
```diff
@@ -85,7 +89,8 @@ install -p dist/%{python_wheelname} -t %{buildroot}%{python_wheeldir}
 
 %check
 rm setup.cfg
-PYTHONPATH=%{buildroot}%{python3_sitelib} py.test-3 -v --ignore build
+%{python3} -m pip install pluggy more-itertools
```
Comment (MateuszMalisz) [inline SPECS/python-wheel/python-wheel.spec:92]: Shouldn't we add these as python3 packages instead?

Comment (mfrw) [reply SPECS/python-wheel/python-wheel.spec:92]: I think, since we have `pip3` available at the test time, it makes the spec file more lean.

Comment (PawelWMS) [reply SPECS/python-wheel/python-wheel.spec:92]: I'd say: if we have them, then yes, but wouldn't bring in a new package for the sake of testing. I believe we do have `python3-pluggy`. I'm not sure, if it builds, though.

**Review by PawelWMS — APPROVED** [review]

File: SPECS/python-wheel/python-wheel.spec (hunk 2 above)
Comment (0xba1a) [reply SPECS/python-wheel/python-wheel.spec:92]: `python3-pluggy` is in SPECS-EXTENDED. Bringing that into core needs additional dependent package movement too. So merging it now until `python3-pluggy` comes into core.

File: SPECS/python-wheel/python-wheel.spec (hunk 1 above)
Comment (0xba1a) [reply SPECS/python-wheel/python-wheel.spec:93]: @MateuszMalisz  `%{python3}` expands to `/usr/bin/python3`. `%{buildroot}%{python3_sitelib}` expands to `/usr/src/mariner/BUILDROOT/python-wheel-0.33.6-12.cm2.x86_64/usr/lib/python3.9/site-packages`.

Let me know if I don't understand your question correctly.

**Review by mfrw — APPROVED** [review]
🚀

File: SPECS/python-wheel/python-wheel.spec (hunk 1 above)
Comment (MateuszMalisz) [reply SPECS/python-wheel/python-wheel.spec:86]: Sounds good, I wasn't sure what the PYTHONPATH points to. This is fine.

**Review by MateuszMalisz — APPROVED** [review]

**Pull request merged**